CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 * [#54](https://github.com/dblock/slack-ruby-bot/pull/54): Improvements to Hook configuration - [@dramalho](https://github.com/dramalho).
 * Your contribution here.
 
+### 0.7.1 (7/7/2016)
```

Comment:
Don't call a release, you're not making a release. This should be under 0.8.1 (Next).